backend/apps/health/views.py

```
@@ -0,0 +1,263 @@
+            
+            # Check if channel layer is configured
+            if channel_layer:
+                result["details"]["channel_layer_type"] = type(channel_layer).__name__
+                result["details"]["response_time_ms"] = round((time.time() - start) * 1000, 2)
+                
+                # Try to create a test group
+                try:
+                    group_name = "health_check_group"
+                    async_to_sync(channel_layer.group_add)(group_name, "health_check_channel")
+                    result["details"]["group_created"] = True
+                except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)